code/model/modelrender.cpp

```diff
@@ -1868,7 +1870,7 @@ void model_render_glowpoint_add_light(int point_num, const vec3d *pos, const mat
 			return;
 	}
 
-	if( (gpo->type_override?gpo->type:bank->type)==0)
+	if( (pmi && gpo->type_override?gpo->type:bank->type)==0)
```

Comment:
Similarly, if `pmi` is nullptr here the `model_instance_local_to_global_dir` function can be skipped because `cone_dir_rot` can be assigned directly to `cone_dir_model`.